src/components/customers/creditNotes/CreditNotesList.tsx

```diff
@@ -264,7 +289,25 @@ CreditNotesList.displayName = 'CreditNotesList'
 
 const SkeletonLine = styled.div`
   display: flex;
-  margin-top: ${theme.spacing(7)};
+  align-items: center;
+  height: ${NAV_HEIGHT}px;
+  justify-content: space-between;
+  box-shadow: ${theme.shadows[7]};
+
+  div {
+    width: 100%;
+    margin-right: 70px;
+    display: flex;
+    justify-content: flex-end;
+
+    span:first-child {
+      margin-right: 16px;
```

Comment:
use the `theme.spacing` here (same for the margin right before, 70 shouldn't be possible 🤔

Reply:
I messed up with the skeleton here. Took a different approach but still had to have a big value on the margin-right.

Used theme spacing and code is simpler. Let me know what you think